src/stk_mcp/cli.py

```
@@ -0,0 +1,99 @@
+
+@app.command()
+def run(
+    host: str = typer.Option("127.0.0.1", help="The host to bind the server to."),
+    port: int = typer.Option(8765, help="The port to run the server on."),
+    mode: StkMode = typer.Option(
+        StkMode.ENGINE if os.name != "nt" else StkMode.DESKTOP,
+        "--mode", "-m",
+        case_sensitive=False,
+        help="STK execution mode. 'desktop' is only available on Windows.",
+        callback=_validate_desktop_mode,
+    ),
```

Ruff (0.12.2)

Do not perform function call `typer.Option` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)